training/tests/integration/conftest.py

```diff
@@ -0,0 +1,46 @@
+# (C) Copyright 2025 Anemoi contributors.
+#
+# This software is licensed under the terms of the Apache Licence Version 2.0
+# which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# In applying this licence, ECMWF does not waive the privileges and immunities
+# granted to it by virtue of its status as an intergovernmental organisation
+# nor does it submit to any jurisdiction.
+
+
+from pathlib import Path
+
+import pytest
+from hydra import compose
+from hydra import initialize
+from omegaconf import OmegaConf
+
+
+@pytest.fixture(
+    params=[
+        ["model=gnn"],
+        ["model=graphtransformer"],
+    ],
```

Comment:
Related to this, what's this conftest for? is this used to permutate the options tested in the `test_training_cycle.py` file? Might be worth mentioning this in the docs?